lib/polymer-atom.js

```diff
@@ -49,6 +49,17 @@ function atomPolylint() {
           });
         });
         return errors;
+      }).catch(function(err){
+        console.log(Object.getOwnPropertyNames(err));
+        if (err.sourceDocument) {
```

Comment:
what kind of errors don't have sourceDocument? If these are unhandled errors in Hydrolysis (as opposed to a handled parse error), then maybe you should re-throw them, or at least console.error().